File: DSA/Ananya_Gambhir/Day12_ques1.java
import java.util.Scanner;
public class Day12_ques2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int number = scanner.nextInt();
        int result = recursiveDigitSum(number);
        System.out.println("Output: " + result);
        scanner.close();
    }
    public static int recursiveDigitSum(int number) {
        if (number < 10) {
            return number;
        }
        int sum = 0;
        while (number > 0) {
            sum += number % 10;
            number /= 10;
        }
        
        return recursiveDigitSum(sum);
    }
}

File: DSA/Umang-Singhal/Day12_Ques1.c
#include <stdio.h>
int fibonacci(int n) {
    if (n <= 0) {
        return 0;
    } else if (n == 1) {
        return 1; 
    } else {
        return fibonacci(n - 1) + fibonacci(n - 2); 
    }
}
int main() {
    int n;
    printf("Enter the value of n: ");
    scanf("%d", &n);
    printf("The %d-th Fibonacci number is: %d\n", n, fibonacci(n));
    return 0;
}
